api-docs/rst/dev-guide/api-operations/methods/get-get-scaling-group-state-v1.0-tenantid-groups-groupid-state.rst

```diff
@@ -14,14 +14,17 @@ This operation retrieves the current state of a scaling group.
 The *GroupState* object consists of the following properties:
 
 
-
-*  *paused*. Specifies whether execution of scaling policies for the group is currently suspended. If this value is set to true, the group will not scale up or down. All policy execution calls are ignored while this value is set to true.
+*  *paused*. Specifies whether execution of scaling policies for the group is currently suspended. If this value
+   is set to TRUE, the group will not scale up or down. All scheduled and API-generated policy executions are suspended, and convergence will
+   not be triggered while this value is set to TRUE. Any *POST* calls to the :ref:`/converge <post-create-scaling-group-v1.1-tenantid-converge>` operation and any *POST* calls to the :ref:`/execute <post-execute-policy-v1.0-tenantid-groups-groupid-policies-policyid-execute>`
+   operation will error with a `403GroupPausedError` error message if convergence is paused. If this value is set FALSE, all scaling and convergence activities, as well as all policy execution calls are resumed.
 *  *pendingCapacity*. Integer. Specifies the number of servers that are in a "building" state.
 *  *name*. Specifies the name of the group.
 *  *active*. Specifies an array of active servers in the group. This array includes the server Id, as well as other data.
 *  *activeCapacity*. Integer. Specifies the number of active servers in the group.
-*  *desiredCapacity*. Integer. Specifies the sum of all servers that are in ``Active`` and ``Pending`` state.
-*  A request to create multiple policies, followed by the matching response.
+*  *desiredCapacity*. Integer. Specifies the number of servers that are desired in the scaling group.
+*  *status*. String. This value can be set to either ACTIVE or ERROR. If *status* is set to ACTIVE, the scaling group is converging, if *status* is set to ERROR, it implies that Autoscale has stopped converging due to an irrecoverable error.
```

Comment:
*status* Shows the current state of convergence activities. If ``status=ACTIVE``, the scaling group is converging. If ``status=ERROR``, convergence activities have stopped, typically due to an unrecoverable error.